CommonCompilerConfig.cmake

```diff
@@ -170,20 +158,13 @@ if(MSVC)
 							$<$<CONFIG:Release>:/O2 /Oi /Gy  /GL /MD> 
 							$<$<CONFIG:Debug>:/Ob0 /Od /Zi /sdl /RTC1 /MDd>)
 	#linker flags
-	if("${BITNESS}" STREQUAL "32")
-		set(machine "/MACHINE:X86")
-	else()
-		set(machine "/MACHINE:X${BITNESS}")
```

Comment:
I think we still need this.  64bit is not always the default on Windows.  I agree that other places we would break if we compiled for 32bit but perhaps we do want to support 32bit at some point.

Reply:
Do I need to revert and use NTA_ARCH_32,... or this block and $BITNESS should be enough for Win? 

>  but perhaps we do want to support 32bit at some point. 

reallistically I don't think so, major distros are dropping 32bit support (Ubuntu, fedora)

Reply:
I see postings in the forums talking about trying to get the nupic library to run on 32bit Linux so it can run on raspberry PI.  That makes since because building a system with 100 raspberry PI modules is a cheap way to get lots of threads running in parallel.
If it does not affect our 64bit code performance I would like to also be able to compile as 32bit for this purpose if we can.

Reply:
I'll keep nta_arch where it makes sence